deployment/sriov-network-operator/templates/leader_election_role.yaml

```diff
@@ -0,0 +1,41 @@
+# permissions to do leader election.
```

Comment:
i dont think this is used today when running `make deploy-k8s`

IMO you need to update the role.yaml template

https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/348e699badfaf15d30fe2c68bfd1f07a4b0e0c10/deploy/role.yaml#L86

perhaps other `.yaml` files from [deploy ](https://github.com/k8snetworkplumbingwg/sriov-network-operator/tree/master/deploy) folder are out of sync

Reply:
Good catch, I'll update my PR with it